### PR TITLE
CSI-1848 update READMEs

### PR DESCRIPTION
**Pull request by oriyarde**
for 1.3.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,23 +3,24 @@ The Container Storage Interface (CSI) Driver for IBM block storage systems enabl
...
 
-Supported operating systems:
-  - RHEL 7.x (x86 architecture)
-  - RHCOS (x86, IBM Z, and IBM PowerPC architectures)
+Supported operating systems (and architectures):
+  - RHEL 7.x (x86)
+  - RHCOS (x86, IBM Z, and IBM PowerPC)
 
-Full documentation can be found on the [IBM Knowledge Center] (https://www.ibm.com/support/knowledgecenter/SSRQ8T).
+Full documentation can be found on the [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSRQ8T).
```
Comment (rivkap) [inline README.md:23]: I'm curious why you removed the space here?

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -113,22 +113,22 @@ spec:
...
+    Supported container platforms (and architectures):
+      - OpenShift v4.3 (IBM Z and IBM PowerPC)
+      - OpenShift v4.4 (x86 and IBM Z)
+      - OpenShift v4.5 (x86)
+      - Kubernetes v1.17 (x86)
+      - Kubernetes v1.18 (x86)
 
     Supported IBM storage systems:
-      - IBM FlashSystem 9100
       - IBM Spectrum Virtualize Family (including IBM Flash family members built with IBM Spectrum Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R) and IBM SAN Volume Controller (SVC) models SV2, SA2)
```
Comment (rivkap) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml:124]: update to:
IBM Spectrum Virtualize Family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R)

File: README.md [hunk 3]
```diff
@@ -3,23 +3,24 @@ The Container Storage Interface (CSI) Driver for IBM block storage systems enabl
...
-  - Kubernetes v1.18
+Supported container platforms (and architectures):
+  - OpenShift v4.3 (IBM Z and IBM PowerPC)
+  - OpenShift v4.4 (x86 and IBM Z)
+  - OpenShift v4.5 (x86)
+  - Kubernetes v1.17 (x86)
+  - Kubernetes v1.18 (x86)
 
 Supported IBM storage systems:
   - IBM Spectrum Virtualize Family (including IBM Flash family members built with IBM Spectrum Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R) and IBM SAN Volume Controller (SVC) models SV2, SA2)
```
Comment (rivkap) [inline README.md:14]: update to:
IBM Spectrum Virtualize Family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R)

File: deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -113,22 +113,22 @@ spec:
...
+    Supported container platforms (and architectures):
+      - OpenShift v4.3 (IBM Z and IBM PowerPC)
+      - OpenShift v4.4 (x86 and IBM Z)
+      - OpenShift v4.5 (x86)
+      - Kubernetes v1.17 (x86)
+      - Kubernetes v1.18 (x86)
 
     Supported IBM storage systems:
-      - IBM FlashSystem 9100
       - IBM Spectrum Virtualize Family (including IBM Flash family members built with IBM Spectrum Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R) and IBM SAN Volume Controller (SVC) models SV2, SA2)
```
Comment (rivkap) [inline deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml:124]: update to:
IBM Spectrum Virtualize Family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R)

**Review by rivkap — CHANGES_REQUESTED** [review]
A few updates needed because we aren't specifically mentioning the SAN volume controller models anymore.

**Review by ArbelNathan — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (oriyarde) [reply README.md:23]: before:
image

after:
image

File: README.md (hunk 3 above)
Comment (oriyarde) [reply README.md]: done

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml (hunk 2 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml]: done

File: deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml (hunk 4 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml]: done

**Review by rivkap — APPROVED** [review]

**Pull request merged**